tests/init.c

```diff
@@ -0,0 +1,56 @@
+#include <check.h>
+
+#include "../ge.h"
+
+START_TEST(initialitiation_state)
```

Comment:
spelling

Reply:
“Initialitiation” is spelt like this in the original docs… I also kept it like that, do we want to “correct” it, or be more faithful to the manuals?

Reply:
yeah that's also the reason I kept it. I like sticking to the manual